server/views/pages/incidentDetails.njk

```diff
@@ -91,6 +87,48 @@
 				})
 			}}
 
+{{ govukRadios({
+    errorMessage: "discoveryRadioSelected" | findError(errors),
+    idPrefix: 'discoveryRadioSelected',
+    id: 'discoveryRadioSelected',
+    name: 'discoveryRadioSelected',
+    attributes: { "data-qa": "radio-buttons-discovery" },
+    fieldset: {
+        legend: {
+        text: "Was the incident discovered at the same time?",
+        isPageHeading: false,
+        classes: "govuk-fieldset__legend--s"
+        }
+    },
+    items: [
+    {
+      value: "Yes",
+      text: "Yes",
+      checked: data.discoveryRadioSelected == 'Yes'
+    },
+    {
+      value: "No",
+      text: "No",
+      checked: data.discoveryRadioSelected == 'No',
+	  conditional:  {
+        html: dateTimePicker({
+					errors: errors,
```

Comment:
The formatting of the njk might need sorting out a bit, there's quite a lot of tabs/spaces